pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -434,6 +438,13 @@ func (m *cgroupManagerImpl) toResources(resourceConfig *ResourceConfig) *libcont
 			Limit:    uint64(0),
 		})
 	}
+	// only for cgroup v2
+	if resourceConfig.Unified != nil {
```

Comment:
does this need to be guarded with `IsCgroupV2` check?

Reply:
fixed